pkg/realmd/operation.js

```diff
@@ -413,7 +413,9 @@
     function setup() {
         var $ = jQuery;
 
-        var element = $("<a>");
+        var element = $("<span>");
+        var link = $("<a>");
+        element.append(link);
```

Comment:
`update_realm_privileged` still puts `disabled` on element, not on the link.

Reply:
`update_privileged()` itself needs to be modified for that, but fortunately all other usages of this involve buttons (where notifications work on disabled buttons). Fixed locally.